core/sdk-core/src/main/java/software/amazon/awssdk/core/client/config/SdkClientOption.java

```diff
@@ -65,7 +64,6 @@
     /**
      * Whether to calculate the CRC 32 checksum of a message based on the uncompressed data. By default, this is false.
      */
-    @ReviewBeforeRelease("Move this to aws-core module once HttpResponseAdaptingStage is removed")
```

Comment:
I didn't move it to `aws-core` because I thought it would be a backward incompatible change to move it to aws-core by now.